src/js/components/Aggregate.js

```diff
@@ -90,6 +90,7 @@ export default class Aggregate extends Component {
 Aggregate.propTypes = {
   a11yTitleId: PropTypes.string,
   a11yDescId: PropTypes.string,
+  attribute: PropTypes.string.isRequired,
```

Comment:
That's in a Ferret context. Aggregate was changed to be more self consistent, like the changes to Filter. So, Ferret should be changed to map the attribute name to the "name" property.